src/services/formatting/formattingScanner.ts

```diff
@@ -278,5 +280,24 @@ namespace ts.formatting {
             }
             return tokenInfo;
         }
+
+        function skipToEndOf(node: Node): void {
+            scanner.setTextPos(backUpWhitespace());

```

Comment:
I tested and it works without `backUpWhitespace`. Now I'm wondering why I thought it was needed.